int64 task_id
string robot_id
string state

